packages/manager/modules/pci/src/projects/project/failover-ips/failover-ip/edit/edit.routing.js

```diff
@@ -19,6 +16,10 @@ export default /* @ngInject */ ($stateProvider) => {
       },
       resolve: {
         breadcrumb: () => null,
+        serviceName: /* @ngInject */ $stateParams => $stateParams.serviceName,
```

Comment:
Shouldn't be `$transition$.params().serviceName` instead?

Reply:
Tiny reminder that `$stateParams` is deprecated ;)

https://ui-router.github.io/ng1/docs/latest/modules/injectables.html#_stateparams

Reply:
thanks for pointing that, i wasn't aware of stateParams being deprecated